src/components/forms/RegisterForm.js

```diff
@@ -23,7 +21,6 @@ class RegisterForm extends Component {
     LocalApi.post("users", { username, email, password })
       .then(response => {
         setAuthToken(response.data.token);
-        //      useHistory().push("/");
```

Comment:
Deleted the unnecessary code & comment here (as we discussed on the previous pull request)

Reply:
ok, thanks